packages/integration-tests/package.json

```diff
@@ -5,8 +5,10 @@
     "license": "Apache-2.0",
     "private": true,
     "scripts": {
-        "build": "node --max_old_space_size=3072 ./node_modules/webpack/bin/webpack.js --mode=development",
-        "postinstall": "yarn rimraf ./node_modules/@0x/mesh-browser/go ./node_modules/@0x/mesh-browser/scripts"
+        "build": "node --max_old_space_size=3072 ./node_modules/.bin/webpack --mode=development",
+        "build:ts": "tsc -b",
+        "postinstall": "yarn rimraf ./node_modules/@0x/mesh-browser/go ./node_modules/@0x/mesh-browser/scripts",
```

Comment:
Can you add a comment to why this is necessary? Since JSON doesn't support comments, our convention is to add a "script" comment like so:

```
"postinstall:comment": "YOUR COMMENT HERE",
"postinstall": "yarn rimraf ./node_modules/@0x/mesh-browser/go ./node_modules/@0x/mesh-browser/scripts",
```